### PR TITLE
TiFlash: Add TiFlash edit-config instruction after manually scaled in TiFlash nodes

### PR DESCRIPTION
**Pull request by birdstorm**
### What is changed, added or deleted? (Required)

Add TiFlash edit-config instruction after manually scaled in TiFlash nodes

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### CODE REVIEW

File: scale-tidb-using-tiup.md [hunk 1]
```diff
@@ -385,6 +385,14 @@ tiup cluster display <cluster-name>
 
 4. 手动删除 TiFlash 的数据文件，具体位置请查看 TiFlash 的配置文件。
```
Comment (JaySon-Huang) [inline scale-tidb-using-tiup.md:386]: "具体位置请查看 TiFlash 的配置文件",
这里补充下配置文件的路径？

Comment (birdstorm) [reply scale-tidb-using-tiup.md:386]: 是在 Topo 中由用户自己配置的，即 Data Dir 目录。

Comment (yikeke) [reply scale-tidb-using-tiup.md:386]: 我改了下 suggestion，看看这样可以吗

Comment (JaySon-Huang) [reply scale-tidb-using-tiup.md:386]: LGTM

Comment (birdstorm) [reply scale-tidb-using-tiup.md]: Thx for your advice, applied.

**Pull request merged**